src/spectrecoff-cli/commands/Status.fs

```diff
@@ -0,0 +1,46 @@
+namespace SpectreCoff.Cli.Commands
+
+open System.Threading.Tasks
+open Spectre.Console
+open Spectre.Console.Cli
+open SpectreCoff
+
+type StatusSettings()  =
+    inherit CommandSettings()
+
+type StatusExample() =
+    inherit Command<StatusSettings>()
+    interface ICommandLimiter<StatusSettings>
+
+    override _.Execute(_context, _settings) =
+        let thinkingProcess (context: StatusContext) =
+            task {
+                do! Task.Delay(5000)
+                context.Status <- "More thinking"
```

Comment:
Maybe we should offer a functions:

```fsharp
updateContext newMessage newSpinner newLook context = 
    context.Status <- newMessage
    context.SpinnerStyle <- newLook |> toSpectreStyle
    context.Spinner <- newSpinner
    context
```

and then go one step further:

```fsharp
type sc = 
  { Message: string
    Spinner: string option
    Look: Look option }
```

which you pass as argument into the start/update method (i.e., have one method which only accepts string like start now, one which takes an instance of this type), and then, on update, if you only want to change the message, you can do 

`context |> update { sc with Message = "Step 2" }`

(Please replace sc with appropriate type name and variable name resp.)

Reply:
This is a great idea :) I will implement it once my pesky real work releases me for a moment :)

Reply:
I changed according to your suggestion. Somewhat unsure about the formatting, but I will not tell you where :) Until then, patiently awaiting your feedback @EluciusFTW